contracts/interfaces/IAxelarAmplifierGateway.sol

```diff
@@ -9,6 +9,32 @@ import { IAxelarGMPGateway } from './IAxelarGMPGateway.sol';
  * @dev Interface for the Axelar Gateway that supports general message passing and contract call execution.
  */
 interface IAxelarAmplifierGateway is IAxelarGMPGateway {
+    /*********\
+    |* Types *|
+    \*********/
+
+    enum CommandType {
+        ApproveContractCall,
+        TransferOperatorship
+    }
+
+    struct Command {
+        CommandType commandType;
+        string messageId;
+        bytes params;
+    }
+
+    struct CommandBatch {
+        bytes32 chainNameHash;
+        bytes32 routerHash;
+        Command[] commands;
+    }
+
+    struct SignedCommandBatch {
+        CommandBatch batch;
+        bytes proof;
```

Comment:
Considering if we should type this as well

Reply:
If we do we better do it now. Before releasing the new multisig.

Reply:
Before you were inclined to keep it compatible with the current gateway, so we changed it [back to bytes](https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/8c0df6bc62d33c5730d04c2155979ad7ae0a7e72/contracts/governance/BaseWeightedMultisig.sol#L77).